<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Memoset</class>
 <widget class="QDialog" name="Memoset">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>739</width>
    <height>401</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QPushButton" name="pushButton_memoset">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>190</y>
     <width>112</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>Set</string>
   </property>
  </widget>
  <widget class="QDateTimeEdit" name="dateTimeEdit_memo">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>100</y>
     <width>194</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_time">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>100</y>
     <width>61</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Time:</string>
   </property>
  </widget>
  <widget class="QLabel" name="labe_matter">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>140</y>
     <width>58</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Matter:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_matter">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>140</y>
     <width>241</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QTextEdit" name="textEdit_matter">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>150</y>
     <width>281</width>
     <height>111</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_todo">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>120</y>
     <width>58</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>To do:</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_showall">
   <property name="geometry">
    <rect>
     <x>400</x>
     <y>90</y>
     <width>112</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>Show all</string>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>60</y>
     <width>300</width>
     <height>33</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QDateEdit" name="dateEdit_searchmemo"/>
    </item>
    <item>
     <widget class="QPushButton" name="pushButton_search">
      <property name="text">
       <string>Search</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QPushButton" name="pushButton_delete">
   <property name="geometry">
    <rect>
     <x>600</x>
     <y>90</y>
     <width>112</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>Delete</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
